File: Fall 2020/9-1/FizzBuzz.java
import java.util.Scanner;
public class FizzBuzz {
    public static void main(String[] args) {
    
    Scanner scan = new Scanner(System.in);
    int n;
    int x;
    int y;
    int v = 1;
    x = scan.nextInt();
    y = scan.nextInt();
    n = scan.nextInt();
    while (!(v > n)) {
        if ((v % x == 0) && (v % y == 0)) {
            System.out.println("FizzBuzz");
            v++;
            }
        else if (v % x == 0) {
            System.out.println("Fizz");
            v++;
            }
        else if (v % y == 0) {
            System.out.println("Buzz");
            v++;
            }
        else {
            System.out.println(v);
            v++;
            }
        }
    }
}

File: Fall 2021/For Fun/FizzBuzz.js
var fizzBuzz = function(n) {
    const output = [];
    for (let i = 0; i < n; i++) {
        if ((i + 1) % 5 == 0 && (i + 1) % 3 == 0) {
            output[i] = "FizzBuzz";
        }
        else if ((i + 1) % 5 == 0) {
            output[i] = "Buzz";
        }
        else if ((i + 1) % 3 == 0) {
            output[i] = "Fizz";
        }
        else {
            output[i] = (i + 1).toString();
        }
    }
    return output;
};